db/seeds.rb

```diff
@@ -112,8 +112,13 @@ def create_claim_description_for_imported_fact_check(user, project, team)
   ClaimDescription.create!(description: Faker::Company.catch_phrase, context: Faker::Lorem.sentence, user: user, project_media: create_blank(project, team))
 end
 
-def create_confirmed_relationship(parent, child)
-  Relationship.create!(source_id: parent.id, target_id: child.id, relationship_type: Relationship.confirmed_type)
+def create_confirmed_relationship(parent, children)
+  if children.class == Array
+    children.each { |child| Relationship.create!(source_id: parent.id, target_id: child.id, relationship_type: Relationship.confirmed_type)}
+  else
+    child = children
+    Relationship.create!(source_id: parent.id, target_id: child.id, relationship_type: Relationship.confirmed_type)
+  end
```

Comment:
If I'm reading this correctly, this method is basically handling the case when there can be one or more children?
If so, I suggestion the following rewrite (just a suggestion, no need to change):

```
[children].flatten.each { |child| Relationship.create!(source_id: parent.id, target_id: child.id, relationship_type: Relationship.confirmed_type) }
```

Reply:
I wondered about this one, will update it ☺️